# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hte/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ht-0.9.1/srt/srt_x86.cc_orig	2006-03-19 21:21:46.410227250 +0100
+++ ht-0.9.1/srt/srt_x86.cc	2006-03-19 21:22:08.743623000 +0100
@@ -76,7 +76,7 @@
 
 class sym_int_reg_x86: public sym_int_reg {
 public:
-	sym_int_reg_x86::sym_int_reg_x86(UINT r): sym_int_reg(r)
+	sym_int_reg_x86(UINT r): sym_int_reg(r)
 	{
 	}
 
